includes/blocks/class-sensei-block-take-course.php

```diff
@@ -55,6 +55,16 @@ public function render_take_course_block( $attributes, $content ): string {
 				Sensei()->notices->add_notice( Sensei()->course::get_course_prerequisite_message( $course_id ), 'info', 'sensei-take-course-prerequisite' );
 				$html = $this->render_disabled( $content );
 			} else {
+				// Replace button label in case it's coming from a sign in with redirect to take course.
```

Comment:
It will be expanded in the future as part of https://github.com/Automattic/sensei/issues/4493